apps/api/src/app/blueprint/usecases/get-grouped-blueprints/get-grouped-blueprints.usecase.ts

```diff
@@ -52,12 +52,12 @@ export class GetGroupedBlueprints {
   ): INotificationTemplate[] {
     const storedBlueprints = this.extractStoredBlueprints(groups);
 
-    const localPopularBlueprints = { ...POPULAR_TEMPLATES_GROUPED };
+    const localPopularIds = [...POPULAR_TEMPLATES_ID_LIST];
 
-    return localPopularBlueprints.blueprints.map((localBlueprint) => {
-      const storedBlueprint = storedBlueprints.find((blueprint) => blueprint._id === localBlueprint._id);
+    return localPopularIds.map((localBlueprintId) => {
+      const storedBlueprint = storedBlueprints.find((blueprint) => blueprint._id === localBlueprintId);
 
-      return (storedBlueprint ? storedBlueprint : localBlueprint) as INotificationTemplate;
+      return (storedBlueprint ? storedBlueprint : localBlueprintId) as INotificationTemplate;
```

Comment:
> wait, something is weird here, this `updatePopularBlueprints` should return the blueprints, but not their `id`'s

You are correctly right, but i refactored a bit the code to return only the ids because at the moment we are not using the objects. If we will add the data to client (through @Shared i will return the metadata.) [REF](https://github.com/novuhq/novu/pull/3502#discussion_r1205391918)



> also it should take into consideration the environment, where we are currently running, because id's from prod won't be the same as id's from dev

I made the adjustment on previous PR :) [REF](https://github.com/novuhq/novu/pull/3500#discussion_r1207004322)